### PR TITLE
Component/input

### PR DESCRIPTION
**Pull request by christophschmitz**
Hello, I worked on my Input component.

- Add Input component
- Add Input to stories
- Add styling to input
- Remove Button component (template)

This pull request resolves: #3 

[Deployment link](https://mastery-component-input-051pm3.herokuapp.com/storybook/?path=/story/component-input--email)

### CODE REVIEW

File: src/app/components/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <div>
+      <label htmlFor={id} className={styles.label}>
+        {label}
+      </label>
+      <input
+        className={styles.input}
+        type={type}
+        value={value}
+        placeholder={placeholder}
+        onChange={onChange}
```
Comment (phlgr) [inline src/app/components/Input/Input.tsx:31]: What happens here onChange? I guess you want to receive the value that is currently in the input field. 
```suggestion
        onChange={event => onChange(event.target.value)}
```

File: src/app/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+export type InputProps = {
+  label: string;
+  id: string;
+  value: string;
+  type: 'email' | 'password';
+  placeholder: string;
+  onChange: () => void;
```
Comment (phlgr) [inline src/app/components/Input/Input.tsx:10]:
```suggestion
  onChange: (value: string) => void;
```

File: src/app/components/Input/Input.module.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+.label {
+  color: var(--text-primary);
+  font-size: 0.75rem;
+  display: block;
+  margin-bottom: 0.3125rem;
+}
+::placeholder {
+  color: var(--text-primary);
+  font-style: italic;
+  font-size: 1rem;
```
Comment (phlgr) [inline src/app/components/Input/Input.module.css:24]: This should not be necessary

File: src/app/components/Input/Input.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  title: 'Component/Input',
+  component: Input,
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+};
+const onChangeFunction = () => {
+  console.log('Hello World');
```
Comment (phlgr) [inline src/app/components/Input/Input.stories.tsx:16]: You could display whatever value is received here
```suggestion
const onChangeFunction = (value: string) => {
  console.log(value);
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/Input/Input.module.css (hunk 3 above)
Comment (christophschmitz) [reply src/app/components/Input/Input.module.css:24]: Should I change it with the Typography component?

**Pull request merged**